lib/compile.dart

```diff
@@ -438,51 +438,44 @@ CompiledFn compileFunction(
         return Bytecode(Bytecode.refFunc, i0: instr.immediate_0 as int);
 
       case Opcode.overflow:
-        switch (instr.opcode2!) {
-          case OverflowOpcode.i32_trunc_sat_f32_s:
-            return Bytecode(Bytecode.i32_trunc_sat_f32_s);
-          case OverflowOpcode.i32_trunc_sat_f32_u:
-            return Bytecode(Bytecode.i32_trunc_sat_f32_u);
-          case OverflowOpcode.i32_trunc_sat_f64_s:
-            return Bytecode(Bytecode.i32_trunc_sat_f64_s);
-          case OverflowOpcode.i32_trunc_sat_f64_u:
-            return Bytecode(Bytecode.i32_trunc_sat_f64_u);
-          case OverflowOpcode.i64_trunc_sat_f32_s:
-            return Bytecode(Bytecode.i64_trunc_sat_f32_s);
-          case OverflowOpcode.i64_trunc_sat_f32_u:
-            return Bytecode(Bytecode.i64_trunc_sat_f32_u);
-          case OverflowOpcode.i64_trunc_sat_f64_s:
-            return Bytecode(Bytecode.i64_trunc_sat_f64_s);
-          case OverflowOpcode.i64_trunc_sat_f64_u:
-            return Bytecode(Bytecode.i64_trunc_sat_f64_u);
-          case OverflowOpcode.memory_init:
-            return Bytecode(Bytecode.memory_init,
-                i0: instr.immediate_0 as int, i1: instr.immediate_1 as int);
-          case OverflowOpcode.data_drop:
-            return Bytecode(Bytecode.data_drop, i0: instr.immediate_0 as int);
-          case OverflowOpcode.memory_copy:
-            return Bytecode(Bytecode.memory_copy,
-                i0: instr.immediate_0 as int, i1: instr.immediate_1 as int);
-          case OverflowOpcode.memory_fill:
-            return Bytecode(Bytecode.memory_fill, i0: instr.immediate_0 as int);
-          case OverflowOpcode.table_init:
-            return Bytecode(Bytecode.table_init,
-                i0: instr.immediate_0 as int, i1: instr.immediate_1 as int);
-          case OverflowOpcode.elem_drop:
-            return Bytecode(Bytecode.elem_drop, i0: instr.immediate_0 as int);
-          case OverflowOpcode.table_copy:
-            return Bytecode(Bytecode.table_copy,
-                i0: instr.immediate_0 as int, i1: instr.immediate_1 as int);
-          case OverflowOpcode.table_grow:
-            return Bytecode(Bytecode.table_grow, i0: instr.immediate_0 as int);
-          case OverflowOpcode.table_size:
-            return Bytecode(Bytecode.table_size, i0: instr.immediate_0 as int);
-          case OverflowOpcode.table_fill:
-            return Bytecode(Bytecode.table_fill, i0: instr.immediate_0 as int);
-
-          default:
-            throw 'unhandled overflow ${instr.opcode2}';
```

Comment:
The cases are exhaustive - this default would never be called?

Reply:
exactly!